src/AppInstallerCommonCore/Filesystem.cpp

```diff
@@ -170,7 +170,7 @@ namespace AppInstaller::Filesystem
 
     bool VerifySymlink(const std::filesystem::path& symlink, const std::filesystem::path& target)
     {
-        const std::filesystem::path& symlinkTargetPath = std::filesystem::read_symlink(symlink);
+        const std::filesystem::path& symlinkTargetPath = std::filesystem::weakly_canonical(symlink);
         return symlinkTargetPath == target;

```

Comment:
the target should be canonicalized before comparing. That should fix the issue
```suggestion
        return symlinkTargetPath == std::filesystem::weakly_canonical(target);

```

Reply:
Hopefully!